cli/cache/common.rs

```diff
@@ -0,0 +1,29 @@
+use deno_core::error::AnyError;
+use deno_runtime::deno_webstorage::rusqlite::Connection;
+
+/// Very fast non-cryptographically secure hash.
+pub fn fast_insecure_hash(bytes: &[u8]) -> u64 {
+  use std::hash::Hasher;
+  use twox_hash::XxHash64;
+
+  let mut hasher = XxHash64::default();
+  hasher.write(bytes);
+  hasher.finish()
+}
+
+/// Runs the common sqlite pragma.
+pub fn run_sqlite_pragma(conn: &Connection) -> Result<(), AnyError> {
+  // Enable write-ahead-logging and tweak some other stuff
+  let initial_pragmas = "
```

Comment:
question: how were these determined?

Reply:
It's from here: https://github.com/denoland/deno/blob/56d0ca7f52d9bff7326bb2b9406ea1e0282a5fbe/ext/webstorage/lib.rs#L69-L81

Divy let me know about these in the last pr: https://github.com/denoland/deno/pull/13462#discussion_r790091554

Reply:
Ok, how were they determined?

Reply:
cc @littledivy -- do you know?